src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -757,6 +762,7 @@ public PreferenceSettings()
             // Default Settings
             IsFirstRun = true;
             IsAnalyticsReportingApproved = true;
+            Locale = Configurations.SupportedLocaleDic.FirstOrDefault().Value;
```

Comment:
Configurations.SupportedLocaleDic can be empty or null?

Reply:
Nope, it is initialized as constants